### PR TITLE
env to from too

### PR DESCRIPTION
**Pull request by emtudo**
Being using env for everything except from.
I believe it is good to use, since you can configure everything via env, if this is the will.

### CODE REVIEW

**Comment by emtudo** [comment]
@GrahamCampbell It is single quotes now!
Thanks!

**Comment by GrahamCampbell** [comment]
You still have changed the phpdoc. Please don't change that.

File: config/mail.php [hunk 1]
```diff
@@ -13,7 +13,7 @@
     |
     | Supported: "smtp", "mail", "sendmail", "mailgun", "mandrill", "ses", "log"
     |
-    */
+     */
```
Comment (GrahamCampbell) [inline config/mail.php]: no

File: config/mail.php [hunk 2]
```diff
@@ -26,7 +26,7 @@
     | applications. A default option is provided that is compatible with
     | the Mailgun mail service which will provide reliable deliveries.
     |
-    */
+     */
```
Comment (GrahamCampbell) [inline config/mail.php]: no

File: config/mail.php [hunk 3]
```diff
@@ -39,7 +39,7 @@
     | users of the application. Like the host we have set this value to
     | stay compatible with the Mailgun e-mail application by default.
     |
-    */
+     */
```
Comment (GrahamCampbell) [inline config/mail.php]: no

File: config/mail.php [hunk 4]
```diff
@@ -52,9 +52,9 @@
     | the same address. Here, you may specify a name and address that is
     | used globally for all e-mails that are sent by your application.
     |
-    */
+     */
```
Comment (GrahamCampbell) [inline config/mail.php]: no

File: config/mail.php [hunk 5]
```diff
@@ -65,7 +65,7 @@
     | the application send e-mail messages. A sensible default using the
     | transport layer security protocol should provide great security.
     |
-    */
+     */
```
Comment (GrahamCampbell) [inline config/mail.php]: no

File: config/mail.php [hunk 6]
```diff
@@ -78,7 +78,7 @@
     | set it here. This will get used to authenticate with your server on
     | connection. You may also set the "password" value below this one.
     |
-    */
+     */
```
Comment (GrahamCampbell) [inline config/mail.php]: etc...

**Comment by emtudo** [comment]
My IDE changes it, thanks!

**Pull request closed without merging**